torch_dftd/nn/base_dftd_module.py

```diff
@@ -145,19 +138,38 @@ def calc_energy_and_forces(
                 results_list[i]["forces"] = forces[batch == i].cpu().numpy()
 
         if cell is not None:
-
             # stress = torch.mm(cell_grad, cell.T) / cell_volume
             # Get stress in Voigt notation (xx, yy, zz, yz, xz, xy)
+            assert isinstance(shift, Tensor)
+            voigt_left = [0, 1, 2, 1, 2, 0]
+            voigt_right = [0, 1, 2, 2, 0, 1]
             if batch is None:
                 cell_volume = torch.det(cell).abs()
-                stress = torch.mm(cell.grad, cell.T) / cell_volume
```

Comment:
There was a bug within this line.
The correct implementation should be `stress = torch.mm(cell.T, cell.grad) / cell_volume` (matrix order).
This can be checked by running test with skewed cell input.